packages/crayons-core/src/components/select-option/select-option.tsx

```diff
@@ -188,8 +188,8 @@ export class SelectOption {
         return (
           <Fragment>
             {checkbox}
-            {this.createIcon()}
-            {this.createConversationDescription()}
+            {this.createConversationIcon()}
```

Comment:
@harshith-venkatesh-freshworks Instead of having a different method for conversation icon, can we add the div with the conversation-icon directly here?
```suggestion
          <div class='conversation-icon'>
            {this.createIcon()}
            </div>
```

Reply:
Sure Rihan, this is good, thanks for the suggestion